benchcab/job_script.py

```diff
@@ -44,7 +41,6 @@ def create_job_script(
 #PBS -q normal
 #PBS -P {project}
 #PBS -j oe
-#PBS -M {email_address}
```

Comment:
Instead should we have:
```
#PBS -m e
```
This will send an email at completion of the script whether it succeeds or fails.